mobile/library/java/org/chromium/net/impl/CronvoyUrlRequestContext.java

```diff
@@ -132,7 +132,8 @@ void setTaskToExecuteWhenInitializationIsCompleted(Runnable runnable) {
       checkHaveAdapter();
       return new CronvoyUrlRequest(this, url, callback, executor, mUserAgent, allowDirectExecutor,
                                    requestAnnotations, trafficStatsTagSet, trafficStatsTag,
-                                   trafficStatsUidSet, trafficStatsUid, requestFinishedListener);
+                                   trafficStatsUidSet, trafficStatsUid, requestFinishedListener,
+                                   idempotency == 1);
```

Comment:
can you use or comment-link to ExperimentalUrlRequest.java IDEMPOTENT definition?

Reply:
Done. I updated the code to use the constant from Cronet API.